CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 ### Breaking
 
 - change password-reset url to be consistent with Plone configuration @erral
+- Simplify over the existing Component Registry API. The `component` key has been flattened for simplification and now it's mapped directly to the `component` argument of `registerComponent`. @sneridagh
+
+See https://docs.voltocms.com/upgrade-guide/ for more information.
```

Comment:
```suggestion
See https://6.dev-docs.plone.org/volto/upgrade-guide/index.html for more information.
```

Reply:
I hate to use a temporary URL for this, but I don't know what else we could do. We don't have a proper redirect on `docs.voltocms.com` either.